### ISSUE TITLE
Accessiblity issues

### ISSUE THREAD

**Issue by mariapraetzellis** [DMPRoadmap/roadmap#3211]
CDL's UX team recently conducted an accessibility audit on the DMPTool. They used the Voluntary Product Accessibility Template (VPAT). @briri is working on addressing the "major" assessability issues uncovered in this audit. This work includes the following:

- [ ] [Tab order problems on admin pages](https://github.com/cdluc3/dmptool/issues/371)
- [ ] [Tab order in write plan](https://github.com/cdluc3/dmptool/issues/373)
- [ ] [Cannot access individual expand and collapse buttons in write plan](https://github.com/cdluc3/dmptool/issues/375) 
- [ ] [Comments & Guidance column collapse is not in tab order](https://github.com/cdluc3/dmptool/issues/374)
- [ ] [Missing form label on user profile page](https://github.com/cdluc3/dmptool/issues/370)
- [ ] [Form control does not have a label, but has a title](https://github.com/cdluc3/dmptool/issues/369)
- [ ] [Missing form label - Create Plan](https://github.com/cdluc3/dmptool/issues/368)
- [ ] [Missing form label [test checkbox]](https://github.com/cdluc3/dmptool/issues/367)
- [ ] [Missing form labels- Contact Us](https://github.com/cdluc3/dmptool/issues/366)
